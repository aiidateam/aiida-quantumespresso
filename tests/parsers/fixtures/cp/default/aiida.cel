      1  0.00007257
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      2  0.00014513
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      3  0.00021770
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      4  0.00029027
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      5  0.00036283
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      6  0.00043540
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      7  0.00050797
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      8  0.00058053
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
      9  0.00065310
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
     10  0.00072567
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000
