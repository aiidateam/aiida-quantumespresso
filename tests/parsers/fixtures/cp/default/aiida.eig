  STEP:      1       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
   31.79   35.51   46.15   53.94
  STEP:      2       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
   27.59   31.10   41.63   50.68
  STEP:      3       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
   22.33   25.45   35.87   46.64
  STEP:      4       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
   17.14   19.69   29.77   42.45
  STEP:      5       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
   12.78   14.63   23.88   38.46
  STEP:      6       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
    9.55   10.74   18.61   34.84
  STEP:      7       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
    7.45    8.16   14.24   31.63
  STEP:      8       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
    6.25    6.74   10.97   28.79
  STEP:      9       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
    5.74    5.99    9.04   26.28
  STEP:     10       0.00
   Eigenvalues (eV), kp =   1 , spin =  1
    5.15    6.01    8.46   24.02
