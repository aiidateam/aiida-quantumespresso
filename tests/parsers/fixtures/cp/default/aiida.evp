#   nfi    time(ps)        ekinc        T_cell(K)     Tion(K)          etot               enthal               econs               econt          Volume        Pressure(GPa)        EXX               EVDW
      1  7.256653E-05  3.436858E-01  0.000000E+00  0.000000E+00          3.98558847          3.98558847          3.98558847          4.32927424  2.656549E+02      -0.00000
      2  1.451331E-04  9.667654E-01  0.000000E+00  0.000000E+00          3.08755201          3.08755201          3.08755201          4.05431736  2.656549E+02      -0.00000
      3  2.176996E-04  1.707442E+00  0.000000E+00  0.000000E+00          1.79358149          1.79358149          1.79358149          3.50102300  2.656549E+02      -0.00000
      4  2.902661E-04  2.391118E+00  0.000000E+00  0.000000E+00          0.26880457          0.26880457          0.26880457          2.65992241  2.656549E+02      -0.00000
      5  3.628326E-04  2.901277E+00  0.000000E+00  0.000000E+00         -1.32073449         -1.32073449         -1.32073449          1.58054234  2.656549E+02      -0.00000
      6  4.353992E-04  3.176072E+00  0.000000E+00  0.000000E+00         -2.82798368         -2.82798368         -2.82798368          0.34808793  2.656549E+02      -0.00000
      7  5.079657E-04  3.206193E+00  0.000000E+00  0.000000E+00         -4.14531151         -4.14531151         -4.14531151         -0.93911888  2.656549E+02      -0.00000
      8  5.805322E-04  3.026046E+00  0.000000E+00  0.000000E+00         -5.21561573         -5.21561573         -5.21561573         -2.18957000  2.656549E+02      -0.00000
      9  6.530988E-04  2.697539E+00  0.000000E+00  0.000000E+00         -6.03008800         -6.03008800         -6.03008800         -3.33254933  2.656549E+02      -0.00000
     10  7.256653E-04  2.291610E+00  0.000000E+00  0.000000E+00         -6.61628032         -6.61628032         -6.61628032         -4.32467025  2.656549E+02      -0.00000
