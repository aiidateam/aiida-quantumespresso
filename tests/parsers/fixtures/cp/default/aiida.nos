      1  7.256653E-05      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      2  1.451331E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      3  2.176996E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      4  2.902661E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      5  3.628326E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      6  4.353992E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      7  5.079657E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      8  5.805322E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
      9  6.530988E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
     10  7.256653E-04      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000      0.00000000
