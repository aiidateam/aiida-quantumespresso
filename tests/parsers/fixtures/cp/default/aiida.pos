      1  0.00007257
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      2  0.00014513
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      3  0.00021770
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      4  0.00029027
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      5  0.00036283
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      6  0.00043540
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      7  0.00050797
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      8  0.00058053
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
      9  0.00065310
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
     10  0.00072567
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01
