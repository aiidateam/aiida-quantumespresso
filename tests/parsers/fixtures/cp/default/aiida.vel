      1  0.00007257
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      2  0.00014513
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      3  0.00021770
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      4  0.00029027
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      5  0.00036283
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      6  0.00043540
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      7  0.00050797
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      8  0.00058053
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
      9  0.00065310
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     10  0.00072567
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
     0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
