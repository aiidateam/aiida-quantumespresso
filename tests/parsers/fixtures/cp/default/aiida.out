
     Program CP v.6.0 (svn rev. 13079) starts on  7May2019 at 14:46: 9

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org",
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors
     R & G space division:  proc/nbgrp/npool/nimage =       4

   Dumping input parameters

     Reading input from aiida.in

   Job Title: MD Simulation


   Atomic Pseudopotentials Parameters
   ----------------------------------

   Reading pseudopotential for specie #  1 from file :
   ./pseudo/Si.pbe-n-rrkjus_psl.1.0.0.UPF
   file type is UPF v.2


   Main Simulation Parameters (from input)
   ---------------------------------------
   Restart Mode       =      -1   from_scratch
   Number of MD Steps =      10
   Print out every            1 MD Steps
   Reads from unit    =      50
   Writes to unit     =      51
   MD Simulation time step            =       3.00
   Electronic fictitious mass (emass) =     400.00
   emass cut-off                      =       3.00

   Simulation Cell Parameters (from input)
   external pressure       =            0.00 [KBar]
   wmass (calculated)      =         7780.83 [AU]
   initial cell from CELL_PARAMETERS card
       0.00000000    2.70000000    2.70000000
       2.70000000    0.00000000    2.70000000
       2.70000000    2.70000000    0.00000000
   ibrav =    0
   alat  =     7.21568608
   a1    =     0.00000000    5.10226056    5.10226056
   a2    =     5.10226056    0.00000000    5.10226056
   a3    =     5.10226056    5.10226056    0.00000000

   b1    =    -0.09799578    0.09799578    0.09799578
   b2    =     0.09799578   -0.09799578    0.09799578
   b3    =     0.09799578    0.09799578   -0.09799578
   omega =     265.65493920

   Energy Cut-offs
   ---------------
   Ecutwfc =   30.0 Ry,      Ecutrho =  240.0 Ry,      Ecuts =  120.0 Ry
   Gcutwfc =    6.3     ,    Gcutrho =   17.8          Gcuts =   12.6
   NOTA BENE: refg, mmx =   0.050000  5760
   Eigenvalues calculated without the kinetic term contribution
   Orthog. with lagrange multipliers : eps =   0.10E-08,  max = 300
   verlet algorithm for electron dynamics
   with friction frice =  0.1000 , grease =  1.0000
   Electron dynamics : the temperature is not controlled
   initial random displacement of el. coordinates with  amplitude=  0.020000

   Electronic states
   -----------------
   Number of Electron =     8, of States =     4
   Occupation numbers :
   2.00 2.00 2.00 2.00


   Exchange and correlations functionals
   -------------------------------------
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)


   Ions Simulation Parameters
   --------------------------
   Ions are not allowed to move
   Ionic position (from input)
   sorted by specie, and converted to real a.u. coordinates
   Species   1 atoms =    2 mass =     51195.82 (a.u.),        28.09 (amu) rcmax =   0.50 (a.u.)
        0.000000     0.000000     0.000000
        2.551130     2.551130     2.551130
   Ionic position read from input file


   Cell Dynamics Parameters (from STDIN)
   -------------------------------------
   Starting cell generated from CELLDM
   Constant VOLUME Molecular dynamics
   cell parameters are not allowed to move

   Verbosity: iverbosity =  0



   Simulation dimensions initialization
   ------------------------------------

   unit vectors of full simulation cell
   in real space:                         in reciprocal space (units 2pi/alat):
   1     0.0000    5.1023    5.1023             -0.7071    0.7071    0.7071
   2     5.1023    0.0000    5.1023              0.7071   -0.7071    0.7071
   3     5.1023    5.1023    0.0000              0.7071    0.7071   -0.7071

     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         214     108     26                 4155     1488     180
     Max         216     109     28                 4158     1492     182
     Sum         859     433    109                16625     5961     725


   Real Mesh
   ---------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    36    36    36      36    36     9       1     1     4
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     36    36    36
   Local number of cell to store the grid ( nrxx ) =      11664
   Number of x-y planes for each processors:
   nr3l =     9    9    9    9

   Smooth Real Mesh
   ----------------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    25    25    25      25    25     7       1     1     4
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     25    25    25
   Local number of cell to store the grid ( nrxx ) =       4375
   Number of x-y planes for each processors:
   nr3sl =     7    6    6    6

   Small Box Real Mesh
   -------------------
   Global Dimensions   Local  Dimensions   Processor Grid
   .X.   .Y.   .Z.     .X.   .Y.   .Z.     .X.   .Y.   .Z.
    24    24    24      24    24    24       1     1     1
   Array leading dimensions ( nr1x, nr2x, nr3x )   =     24    24    24
   Local number of cell to store the grid ( nrxx ) =      13824

   unit vectors of box grid cell
   in real space:                         in reciprocal space:
       0.0000    3.4015    3.4015             -0.7071    0.7071    0.7071
       3.4015    0.0000    3.4015              0.7071   -0.7071    0.7071
       3.4015    3.4015    0.0000              0.7071    0.7071   -0.7071

   Reciprocal Space Mesh
   ---------------------
   Large Mesh
     Global(ngm_g)    MinLocal       MaxLocal      Average
           8313           2078           2079        2078.25
   Smooth Mesh
     Global(ngms_g)   MinLocal       MaxLocal      Average
           2981            744            746         745.25
   Wave function Mesh
     Global(ngw_g)    MinLocal       MaxLocal      Average
            363             90             91          90.75

   Small box Mesh
   ngb =         2479 not distributed to processors


   System geometry initialization
   ------------------------------
   ibrav =    0       cell parameters read from input file

   unit vectors of box grid cell
   in real space:                         in reciprocal space:
       0.0000    3.4015    3.4015             -0.7071    0.7071    0.7071
       3.4015    0.0000    3.4015              0.7071   -0.7071    0.7071
       3.4015    3.4015    0.0000              0.7071    0.7071   -0.7071

   Matrix Multiplication Performances
   ortho mmul, time for parallel driver      =   0.00000 with    1 procs

   Constraints matrixes will be distributed block like on
   ortho sub-group =    1*   1 procs



   Pseudopotentials initialization
   -------------------------------
  nlinit  nh(is), ngb, is, kkbeta, lmaxq =           18        2479           1         829           5

                        qqq
  -0.0772  -0.0666   0.0000   0.0000   0.0000   0.0000
  -0.0666  -0.0419   0.0000   0.0000   0.0000   0.0000
   0.0000   0.0000  -0.0095  -0.0249   0.0000   0.0000
   0.0000   0.0000  -0.0249  -0.0561   0.0000   0.0000
   0.0000   0.0000   0.0000   0.0000   0.0059   0.0065
   0.0000   0.0000   0.0000   0.0000   0.0065   0.0070



   Common initialization

   Specie:     1
   1  indv=  1   ang. mom=  0
   2  indv=  2   ang. mom=  0
   3  indv=  3   ang. mom=  1
   4  indv=  3   ang. mom=  1
   5  indv=  3   ang. mom=  1
   6  indv=  4   ang. mom=  1
   7  indv=  4   ang. mom=  1
   8  indv=  4   ang. mom=  1
   9  indv=  5   ang. mom=  2
  10  indv=  5   ang. mom=  2
  11  indv=  5   ang. mom=  2
  12  indv=  5   ang. mom=  2
  13  indv=  5   ang. mom=  2
  14  indv=  6   ang. mom=  2
  15  indv=  6   ang. mom=  2
  16  indv=  6   ang. mom=  2
  17  indv=  6   ang. mom=  2
  18  indv=  6   ang. mom=  2

                        dion
   0.2480   0.2226   0.0000   0.0000   0.0000   0.0000
   0.2226   0.5178   0.0000   0.0000   0.0000   0.0000
   0.0000   0.0000   0.0832   0.0571   0.0000   0.0000
   0.0000   0.0000   0.0571   0.1029   0.0000   0.0000
   0.0000   0.0000   0.0000   0.0000  -0.0224  -0.0221
   0.0000   0.0000   0.0000   0.0000  -0.0221  -0.0218

   Cell parameters from input file are used in electron mass preconditioning
   init_tpiba2=    0.75823613

   Short Legend and Physical Units in the Output
   ---------------------------------------------
   NFI    [int]          - step index
   EKINC  [HARTREE A.U.] - kinetic energy of the fictitious electronic dynamics
   TEMPH  [K]            - Temperature of the fictitious cell dynamics
   TEMP   [K]            - Ionic temperature
   ETOT   [HARTREE A.U.] - Scf total energy (Kohn-Sham hamiltonian)
   ENTHAL [HARTREE A.U.] - Enthalpy ( ETOT + P * V )
   ECONS  [HARTREE A.U.] - Enthalpy + kinetic energy of ions and cell
   ECONT  [HARTREE A.U.] - Constant of motion for the CP lagrangian



   Wave Initialization: random initial wave-functions
   Occupation number from init
   nbnd =     4
    2.00 2.00 2.00 2.00

   formf: eself=    25.53231
   formf:     vps(g=0)=   0.0259822     rhops(g=0)=  -0.0140240
   formf: sum_g vps(g)=   2.4288184 sum_g rhops(g)=  -2.8759882
   Delta V(G=0):   0.023652Ry,    0.643595eV

   from rhoofr: total integrated electronic density
   in g-space =      8.000000   in r-space =     8.000000

  nfi     ekinc              temph  tempp     etot                 enthal               econs                econt              vnhh    xnhh0   vnhp    xnhp0
      1    0.343685770070843    0.0    0.00       3.985588469156       3.985588469156       3.985588469156       4.329274239227   0.0000   0.0000   0.0000   0.0000
      2    0.966765350754185    0.0    0.00       3.087552011428       3.087552011428       3.087552011428       4.054317362182   0.0000   0.0000   0.0000   0.0000
      3    1.707441510496073    0.0    0.00       1.793581493294       1.793581493294       1.793581493294       3.501023003791   0.0000   0.0000   0.0000   0.0000
      4    2.391117837015249    0.0    0.00       0.268804574857       0.268804574857       0.268804574857       2.659922411872   0.0000   0.0000   0.0000   0.0000
      5    2.901276825223175    0.0    0.00      -1.320734486098      -1.320734486098      -1.320734486098       1.580542339125   0.0000   0.0000   0.0000   0.0000
      6    3.176071604444053    0.0    0.00      -2.827983678460      -2.827983678460      -2.827983678460       0.348087925984   0.0000   0.0000   0.0000   0.0000
      7    3.206192634070005    0.0    0.00      -4.145311511098      -4.145311511098      -4.145311511098      -0.939118877028   0.0000   0.0000   0.0000   0.0000
      8    3.026045730253000    0.0    0.00      -5.215615732887      -5.215615732887      -5.215615732887      -2.189570002634   0.0000   0.0000   0.0000   0.0000
      9    2.697538668990559    0.0    0.00      -6.030087998113      -6.030087998113      -6.030087998113      -3.332549329122   0.0000   0.0000   0.0000   0.0000

 * Physical Quantities at step:    10

   from rhoofr: total integrated electronic density
   in g-space =      8.000000   in r-space =     8.000000


                total energy =       -6.61628031989 Hartree a.u.
              kinetic energy =        7.63201 Hartree a.u.
        electrostatic energy =       -9.14022 Hartree a.u.
                         esr =        0.00000 Hartree a.u.
                       eself =       25.53231 Hartree a.u.
      pseudopotential energy =        0.58627 Hartree a.u.
  n-l pseudopotential energy =        0.44545 Hartree a.u.
 exchange-correlation energy =       -6.13979 Hartree a.u.
           average potential =        0.00000 Hartree a.u.



   Eigenvalues (eV), kp =   1 , spin =  1

    5.15    6.01    8.46   24.02

   Allocated memory (kb) =     19580

   CELL_PARAMETERS
    0.00000000    5.10226056    5.10226056
    5.10226056    0.00000000    5.10226056
    5.10226056    5.10226056    0.00000000

   System Density [g/cm^3] :              2.3694143949


   System Volume [A.U.^3] :            265.6549391979


   Center of mass square displacement (a.u.):   0.000000

   ATOMIC_POSITIONS
   Si      0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
   Si      0.25511302793956E+01     0.25511302793956E+01     0.25511302793956E+01

   ATOMIC_VELOCITIES
   Si      0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00
   Si      0.00000000000000E+00     0.00000000000000E+00     0.00000000000000E+00



   Partial temperatures (for each ionic specie)
   Species  Temp (K)   Mean Square Displacement (a.u.)
        1   0.00E+00     0.0000E+00

  nfi     ekinc              temph  tempp     etot                 enthal               econs                econt              vnhh    xnhh0   vnhp    xnhp0
     10    2.291610069896371    0.0    0.00      -6.616280319893      -6.616280319893      -6.616280319893      -4.324670249997   0.0000   0.0000   0.0000   0.0000

   writing restart file: ./out/aiida_51.save
   restart file written in    0.016 sec.



   Averaged Physical Quantities
                      accumulated      this run
   ekinc         :        2.27077       2.27077 (AU)
   ekin          :       11.47877      11.47877 (AU)
   epot          :      -13.54168     -13.54168 (AU)
   total energy  :       -1.70205      -1.70205 (AU)
   temperature   :        0.00000       0.00000 (K )
   enthalpy      :       -1.70205      -1.70205 (AU)
   econs         :       -1.70205      -1.70205 (AU)
   pressure      :        0.00000       0.00000 (Gpa)
   volume        :      265.65494     265.65494 (AU)



     Called by MAIN_LOOP:
     initialize   :      4.71s CPU      4.74s WALL (       1 calls)
     main_loop    :      0.68s CPU      0.69s WALL (      10 calls)
     cpr_total    :      0.70s CPU      0.70s WALL (       1 calls)

     Called by INIT_RUN:

     Called by CPR:
     cpr_md       :      0.70s CPU      0.70s WALL (      10 calls)
     move_electro :      0.67s CPU      0.68s WALL (      10 calls)

     Called by move_electrons:
     rhoofr       :      0.03s CPU      0.04s WALL (      11 calls)
     vofrho       :      0.10s CPU      0.10s WALL (      11 calls)
     dforce       :      0.00s CPU      0.00s WALL (      22 calls)
     calphi       :      0.00s CPU      0.00s WALL (      11 calls)
     newd         :      0.60s CPU      0.60s WALL (      11 calls)

     Called by ortho:
     ortho_iter   :      0.00s CPU      0.00s WALL (      11 calls)
     rsg          :      0.00s CPU      0.00s WALL (      11 calls)
     rhoset       :      0.00s CPU      0.00s WALL (      11 calls)
     sigset       :      0.00s CPU      0.00s WALL (      11 calls)
     tauset       :      0.00s CPU      0.00s WALL (      11 calls)
     ortho        :      0.00s CPU      0.00s WALL (      11 calls)
     updatc       :      0.00s CPU      0.00s WALL (      11 calls)

     Small boxes:
     rhov         :      0.03s CPU      0.03s WALL (      11 calls)
     fftb         :      0.46s CPU      0.46s WALL (    3808 calls)
     set_cc       :      0.00s CPU      0.01s WALL (      11 calls)

     Low-level routines:
     prefor       :      0.00s CPU      0.00s WALL (      11 calls)
     nlsm1        :      0.00s CPU      0.00s WALL (      34 calls)
     fft          :      0.04s CPU      0.04s WALL (     110 calls)
     ffts         :      0.00s CPU      0.00s WALL (      22 calls)
     fft_scatter  :      0.02s CPU      0.02s WALL (     198 calls)
     betagx       :      2.68s CPU      2.68s WALL (       1 calls)
     qradx        :      1.33s CPU      1.33s WALL (       1 calls)
     gram         :      0.00s CPU      0.00s WALL (       1 calls)
     nlinit       :      4.59s CPU      4.61s WALL (       1 calls)
     init_dim     :      0.00s CPU      0.00s WALL (       1 calls)
     newnlinit    :      0.07s CPU      0.08s WALL (       1 calls)
     from_scratch :      0.10s CPU      0.11s WALL (       1 calls)
     strucf       :      0.00s CPU      0.00s WALL (       1 calls)
     calbec       :      0.00s CPU      0.00s WALL (      12 calls)
     ALLTOALL     :      0.00s CPU      0.00s WALL (      66 calls)


     CP           :     5.61s CPU         5.64s WALL


   This run was terminated on:  14:46:15   7May2019

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
